testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7332,81 +7332,153 @@ private static void checkIf(SqlOperatorFixture f) {
   @Test void testSafeAddFunc() {
     final SqlOperatorFixture f0 = fixture().setFor(SqlLibraryOperators.SAFE_ADD);
     f0.checkFails("^safe_add(2, 3)^",
-        "No match found for function signature "
```

Comment:
Are these new indents supposed to be here?

Reply:
Probably not

Reply:
Sometimes, I encounter the same situation  if I commit by IntelliJ IDEA.

I will manually restore them to avoid damaging the git history.